Contributing.md

```diff
@@ -0,0 +1,115 @@
+# Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+[Code of Conduct](#code-of-conduct) 
+# Pull Request Process
+1.	Fork the repository and create a separate branch regarding your contribution.
+2.	Update the CHANGELOG file with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations, and container parameters.
+3.	Changes to README file should be commited **ONLY** when there are changes relevant to the essence of the project, or the ways to deploy it!
+4.	Issue a pull request and ask us to review it. Don't forget to link the PR to a relevant issue if it exists, **If not CREATE it!**
+5.	Make sure your code is conformed to the PEP8 standard
+
+**Note: If your PR do not meet one of the requirements, the PR will not be reviewed!**
+
+### Issue a pull request
+1.	Set yourself as an Assignee
+2.	Set whoever you want to review your PR as a reviewer (can be multiple)
+3.	Make SMALLER pull requests
+4.	Write useful descriptions and titles
+5.	You may merge the Pull Request once you have the sign-off of two other developers, or if you do not have permission to do that, you may request the second reviewer to merge it for you.
+
+#### Pull request EX:
+1. Making changes/Creating XFILE
+2. Empty line
+3. What?
+4. Why?
+5. How?
+6. Testing?
+7. Screenshots(if needed)
+8. Anything else?
+
+### How to commit properly:
```

Comment:
You can link to the conventional commit guide instead of quoting from it. just make sure you make it easy to understand this project is following this convention and it has to be followed through.
https://www.conventionalcommits.org/en/v1.0.0/